concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1384	1384	die Ordinalia von 1-20	vierte, vierter	fourth		
+Steinthal-1875-1565-1385	1385	die Ordinalia von 1-20	fünfte, fünfter	fifth		
+Steinthal-1875-1565-1386	1386	die Ordinalia von 1-20	sechste, sechster	sixth		
+Steinthal-1875-1565-1387	1387	die Ordinalia von 1-20	siebte, siebter	seventh		
+Steinthal-1875-1565-1388	1388	die Ordinalia von 1-20	acht, achter	eighth		
+Steinthal-1875-1565-1389	1389	die Ordinalia von 1-20	neunte, neunter	ninth		
+Steinthal-1875-1565-1390	1390	die Ordinalia von 1-20	zehnte, zehnter	tenth		
+Steinthal-1875-1565-1391	1391	die Ordinalia von 1-20	elfte, elfter	eleventh		
+Steinthal-1875-1565-1392	1392	die Ordinalia von 1-20	zwölfte, zwölfter	twelfth		
+Steinthal-1875-1565-1393	1393	die Ordinalia von 1-20	dreizehnte, dreizehnter	thirteenth		
+Steinthal-1875-1565-1394	1394	die Ordinalia von 1-20	vierzehnte, vierzehnter	fourteenth		
+Steinthal-1875-1565-1395	1395	die Ordinalia von 1-20	fünfzehnte, fünfzehnter	fifteenth		
+Steinthal-1875-1565-1396	1396	die Ordinalia von 1-20	sechzehnte, sechzehnter	sixteenth		
+Steinthal-1875-1565-1397	1397	die Ordinalia von 1-20	siebzehnte, siebzehnter	seventeenth		
+Steinthal-1875-1565-1398	1398	die Ordinalia von 1-20	achtzehnte, achtzehnter	eighteenth		
+Steinthal-1875-1565-1399	1399	die Ordinalia von 1-20	neunzehnte, neunzehnter	nineteenth		
+Steinthal-1875-1565-1400	1400	die Ordinalia von 1-20	zwanzigste, zwanzigster	twentieth		
+Steinthal-1875-1565-1401	1401	der mittelste	der mittelste	the middlemost		
+Steinthal-1875-1565-1402	1402	der letzte	der letzte	the last	1529	LAST (FINAL)
+Steinthal-1875-1565-1403	1403	"die Cardinalia von 1-10 mit ""Mal"""	einmal	once	374	ONE TIME
+Steinthal-1875-1565-1404	1404	"die Cardinalia von 1-10 mit ""Mal"""	zweimal	twice	1546	TWICE
+Steinthal-1875-1565-1405	1405	"die Cardinalia von 1-10 mit ""Mal"""	dreimal	thrice	417	THREE TIMES
+Steinthal-1875-1565-1406	1406	"die Cardinalia von 1-10 mit ""Mal"""	viermal	four times		
+Steinthal-1875-1565-1407	1407	"die Cardinalia von 1-10 mit ""Mal"""	fünfmal	five times		
+Steinthal-1875-1565-1408	1408	"die Cardinalia von 1-10 mit ""Mal"""	sechsmal	six times		
+Steinthal-1875-1565-1409	1409	"die Cardinalia von 1-10 mit ""Mal"""	siebenmal	seven times		
+Steinthal-1875-1565-1410	1410	"die Cardinalia von 1-10 mit ""Mal"""	achtmal	eight times		
+Steinthal-1875-1565-1411	1411	"die Cardinalia von 1-10 mit ""Mal"""	neunmal	nine times		
+Steinthal-1875-1565-1412	1412	"die Cardinalia von 1-10 mit ""Mal"""	zehnmal	ten times		
+Steinthal-1875-1565-1413	1413	noch einmal	noch einmal	once more		
+Steinthal-1875-1565-1414	1414	Wie oft?	Wie oft?	How often?		
+Steinthal-1875-1565-1415	1415	Wann?	Wann?	When?	1238	WHEN
+Steinthal-1875-1565-1416	1416	früher	früher	earlier		
+Steinthal-1875-1565-1417	1417	vor Alters	vor Alters	erstwhile		
+Steinthal-1875-1565-1418	1418	einst (eines Tages)	einst (eines Tages)	once	2914	ONCE (IN THE PAST)
```

Comment:
"eines Tages" at least means "once in the future".

Reply:
It can be both, no? "Eines Tages geschah es, dass..."
Granted, the past use of this sounds a bit old-fashioned and like something out of a fairytale, but then this list is from 1875. Especially in connection with einst, I'd say we can safely assume the past meaning.